docs/integrations/github_actions/aws_oidc.md

```diff
@@ -0,0 +1,172 @@
+
+### Configuration
+
+Update the `default.tfvars` file for the below variables.
+
+* `github_repo`: GitHub repository that needs the access token. Example: octo-org/octo-repo
+* `github_branch`: GitHub branch that runs the workflow. If you plan to trigger the workflow through schedule, then this must be the default branch. If you plan to run the workflow manually, this can be any branch. Example: master
+* `aws_iam_role_name`: Name of the AWS IAM Role to create. Example: steampipe_gh_oidc_demo
+
+### Implementation
+
+Run these commands to create resources in your AWS Account.
+
+* `terraform init`: Initialize Terraform to get all necessary providers.
+* `terraform apply -var-file=default.tfvars`: Apply the configuration using the configuration file "defaults.tfvars"
+
+### Verification
+
+Terraform creates an Identity provider and IAM Role in your account.
+
+* AWS > IAM > Identity provider > token.actions.githubusercontent.com
+* AWS > IAM > Role (rolename: steampipe_gh_oidc_demo)
+
+<div style={{"marginBottom":"2em","borderWidth":"thin", "borderStyle":"solid", "borderColor":"lightgray", "padding":"20px", "width":"90%"}}>
+<img alt="aws_iam_identity_provider" src="/images/docs/ci-cd-pipelines/oidc/aws_iam_identity_provider.png" />
+</div>
+
+<div style={{"marginBottom":"2em","borderWidth":"thin", "borderStyle":"solid", "borderColor":"lightgray", "padding":"20px", "width":"90%"}}>
+<img alt="aws_iam_role" src="/images/docs/ci-cd-pipelines/oidc/aws_iam_role.png" />
+</div>
+
+You will need to add one GitHub Secret, `OIDC_AWS_ROLE_TO_ASSUME`, which is the ARN of the above IAM role that handles OIDC federation. You can get the IAM Role ARN from the Terraform output or the AWS IAM Console.
+
+<div style={{"marginBottom":"2em","borderWidth":"thin", "borderStyle":"solid", "borderColor":"lightgray", "padding":"20px", "width":"90%"}}>
+<img alt="gh_secret" src="/images/docs/ci-cd-pipelines/oidc/gh_secret.png" />
+</div>
+
+## Running the workflow on-demand
+
+The job will run on schedule, but it's always helpful to [run manually](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow) for sanity-check. Make sure you select the correct branch when executing this manually, this should be listed in the Trust Relationships of your IAM Role. (`github_branch` variable in the Terraform script).
+
+<div style={{"marginBottom":"2em","borderWidth":"thin", "borderStyle":"solid", "borderColor":"lightgray", "padding":"20px", "width":"90%"}}>
+<img alt="manual_run" src="/images/docs/ci-cd-pipelines/oidc/manual_run.png" />
+</div>
+
+Here's the result in the GitHub Actions log.
+
+<div style={{"marginBottom":"2em","borderWidth":"thin", "borderStyle":"solid", "borderColor":"lightgray", "padding":"20px", "width":"90%"}}>
+<img alt="save_gh_run" src="/images/docs/ci-cd-pipelines/oidc/save_gh_run.png" />
+</div>
+
+Here's the resulting HTML file in a local instance of the repository.
```

Comment:
"Heres..."  (see above).  What is the purpose of this screenshot?   Does it provide more information?  Is someone supposed to do something with those files?   What is in the "the resulting HTML file"?  Is that the steampipe check output?